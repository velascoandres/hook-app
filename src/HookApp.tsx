import React from 'react'
import { CounterApp } from './components/01-useState/CounterApp'
import { CounterWithCustomHook } from './components/01-useState/CounterWithCustomHook'
import { FormWithCustomHook } from './components/02-useEffect/FormWithCustomHook'
import { SimpleForm } from './components/02-useEffect/SimpleForm'
import { MultipleCustomHooks } from './components/03-examples/MultipleCustomHooks'
import { FocusScreen } from './components/04-useRef/FocusScreen'
import { ReafExampleRef } from './components/04-useRef/ReafExampleRef'
import { Layout } from './components/05-useLayoutEffect/Layout';
import { CallbackHook } from './components/06-memos/CallbackHook'
import { MemoHook } from './components/06-memos/MemoHook'
import { Memorize } from './components/06-memos/Memorize'
import { TodoApp } from './components/08-useReducer/TodoApp'

export const HookApp = () => {
    return (
        <div>
            {/*
            <CounterApp /> 
            <CounterWithCustomHook /> 
            <SimpleForm />
            <FormWithCustomHook />
            <MultipleCustomHooks />
            <FocusScreen />
            <ReafExampleRef />
            <Layout />
            <Memorize />
            <MemoHook />
            <CallbackHook />
            */}
            <TodoApp />
        </div>
    );
}
