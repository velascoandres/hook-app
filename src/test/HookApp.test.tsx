import { shallow } from 'enzyme';
import {HookApp} from '../HookApp';

describe('Pruebas en <HookApp />', () => {
   test('Debe mostrarse correctamente', () => {
       const wrapper = shallow( <HookApp />);
       expect(wrapper).toMatchSnapshot();
   });
   
});
