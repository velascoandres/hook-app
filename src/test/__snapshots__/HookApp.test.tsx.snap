// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe mostrarse correctamente 1`] = `
<div>
  <TodoApp />
</div>
`;

